src/platform/forms/save-in-progress/SaveInProgressErrorPage.jsx

```diff
@@ -182,14 +180,16 @@ SaveInProgressErrorPage.propTypes = {
   isStartingOver: PropTypes.bool.isRequired,
   // For SignInLink
   isLoggedIn: PropTypes.bool.isRequired,
+  formConfig: PropTypes.object,
```

Comment:
Can we use `.shape` and specify that we'll use an optional `continueAppMessage` string in it? Also, I believe the `formConfig` itself should be `.isRequired`, right?

Reply:
Fixed